executable-spec/src/Cardano/Ledger/Spec/STS/Transaction/Transaction.hs

```diff
@@ -14,23 +14,22 @@ import           Data.Text (Text)
 import           Hedgehog (Gen)
 import qualified Hedgehog.Gen as Gen
 import qualified Hedgehog.Range as Range
-
 import           Control.State.Transition (Embed, Environment, PredicateFailure,
                      STS, Signal, State, TRC (TRC), initialRules,
                      judgmentContext, trans, transitionRules, wrapFailed)
 
 import           Ledger.Core (Slot, (⨃))
 
-data TRANSACTION
-
 -- | Dummy transaction, containing some text.
 newtype Transaction = Transaction { text :: Text }
   deriving (Eq, Ord, Show)
 
 instance HeapWords Transaction where
   heapWords (Transaction text) = heapWords text
 
-data St = St { submitted :: Map Transaction Slot }
+data TRANSACTION
+
+data St = St { txToSlot :: Map Transaction Slot }
```

Comment:
I'm not sure about this name: `txToSlot` makes me think about a function that converts transactions to slots.